spec/request/api/v1/auth/sign_out_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+require 'faker'
```

Comment:
If you have faker included in you `test` namespace in the gemfile should be enough